grails-app/views/_menu/_navbar.gsp

```diff
@@ -58,19 +58,12 @@
                         </g:link>
                     </li>
                     <li class="${controllerName.equals('aggregation') ? 'active' : ''}" id="aggregationMainMenu">
-                        <g:link controller="aggregation" action="show"
+                        <g:link controller="aggregationLegacy" action="show"
                                 title="${message(code:'de.iteratec.isocsi.aggregationChart', default:'Aggregation')}">
                             <i class="fas fa-chart-bar"></i>
                             <g:message code="de.iteratec.isocsi.aggregationChart" default="Aggregation"/>
                         </g:link>
                     </li>
-                    <!--<li class="${request.forwardURI.equals('/aggregationDev/show') ? 'active' : ''}"
-                        data-active-matches="/aggregationDev/?.*">
-                        <a href="${createLink(uri: '/aggregationDev/show')}">
-                            <i class="fas fa-chart-bar"></i>
-                            <g:message message="frontend.de.iteratec.osm.aggregation.title"/>
-                        </a>
-                    </li>-->
```

Comment:
The navigation bar links to the legacy page of the aggregation chart. Is this the desired behavior?

Reply:
Nope. I fixed it and now the navigation works in like in a SPA when we come from another Angular component.